Assets/Scripts/CombatZone.cs

```diff
@@ -63,16 +66,27 @@ private void SetCombatPositions()
 
     public void DestroyCombatZone()
     {
-        // TODO Aramando ;)
-        Debug.Log("TODO by Aramdo");
-    }
+        Debug.Log("Combat has Ended");
 
-    void OnTriggerExit(Collider other)
-    {
-        for (int j = 0; j < PlayerPositions.Length; j++)
+        foreach(GameObject player in Players)
+        {
+            EnableMovement(player);
+        }
+
+        foreach (GameObject enemy in Enemies)
         {
-            Players[j].GetComponent<CharacterController>().enabled = true;
-            Players[j].GetComponent<Player>().enabled = true;
+            enemy.GetComponent<Combatant>().RemoveHealthBars();
+            Destroy(enemy);
         }
+
+        Destroy(gameObject);
+    }
+
+    private static void EnableMovement(GameObject player)
+    {
+        player.GetComponent<CharacterController>().enabled = true;
+        player.GetComponent<Combatant>().RemoveHealthBars();
+        player.GetComponent<Player>().enabled = true;
+        player.GetComponentInChildren<Costume>().DisplayAbilities(false);
```

Comment:
This is good for now, but maybe we'll want later to create proper methods in the `Player` class for enabling/disabling movement + in the `Costume` class for hiding the abilities menu in order to have better decoupling. Also, similar comment to here https://github.com/The-GameDJs/CostumeCurse/pull/31#discussion_r518044369 but right now `EnableMovement` technically does more than enabling movement, so maybe there'd be a better name for this method? Like `SwitchToOverworldControls` or something similar 🤔

Reply:
@jmcduff Yup! I brought this up too, but given we're presenting today, I'm ok with this as is. 

That being said, `CombatZone` is really ripping apart the `Combatant` here lmao

Reply:
TBD after today ;)

Reply:
Again, good suggestion! Did not think of this when programming, only thought of the CombatZone script alone. Thanks!